webpack/assets/javascripts/react_app/common/IntegrationTestHelper.js

```diff
@@ -3,6 +3,7 @@ import { mount } from 'enzyme';
 import { Provider } from 'react-redux';
 import { applyMiddleware, combineReducers, createStore } from 'redux';
 import thunk from 'redux-thunk';
+import { ajaxRequestAction } from '../redux/middlewares/ajaxRequestAction';
```

Comment:
should we rename it as it's not an action anymore?

Reply:
APImiddleware? (Later this middle ware will be used for PUT and POST options as well)

Reply:
+1 for `ApiMiddleware`